e2e/features/stepDefinitions/authentication.steps.ts

```diff
@@ -69,6 +85,50 @@ Given(/^I switch to be a `?(?:Solicitor|Legal Rep)(?:| A)`?$/, async function ()
     await ccdPage.contentContains('Immigration');
 });
 
+Given(/^I switch to be a `?(?:Home Office APC)`?$/, async function () {
+    await browser.sleep(100);
```

Comment:
Seems like lot of duplication here except the signIn part. Could we generify in anyway?

Reply:
Specific to each user as per the existing code base. We could look to refactor at some point in the future.